torch/_dynamo/repro/after_aot.py

```diff
@@ -564,7 +564,7 @@ def save_hook(name, val):
         known_names.add(name)
         if not options.skip_saving_inductor_intermediates:
             writer.write_tensor(os.path.join("inductor", name), val)
-        pbar.update(1)
+        pbar.update(1)  # type: ignore[has-type]
```

Comment:
I'm not really sure why enabling typechecking for debug.py is causing mypy to start raising an error here

Reply:
some of the files may be opted into mypy following imports instead of no-follow.. not sure